CONTRIBUTING.md

```diff
@@ -23,12 +23,10 @@ Thanks for your interest in contributing to Ionic's documentation! :tada: Check
 In order to run the documentation locally, install the dependencies and run the development server:
 
 ```sh
-$ npm install --legacy-peer-deps
-$ npm start
+npm install
+npm start
 ```
 
-> **Note**: certain versions of npm (5-8) and Node.js (10-16) are required to run certain scripts.
-
 ---
 
 ## Using VS Code on Windows
```

Comment:
The page on mobile goes outside the viewport due to the header:

| main | branch |
| - | - |
| ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/4f50e935-4c97-4440-a31a-396c3d5989a9) | ![image](https://github.com/ionic-team/ionic-docs/assets/2721089/df24d81a-cea6-4110-a41f-cd0ff2815db5) |